ui/.npmrc

```diff
@@ -1,3 +1,3 @@
-@fortawesome:registry=https://npm.fontawesome.com/
+@fortawesome:registry=https://registry.yarnpkg.com/
```

Comment:
We're seeing "401 Unauthorized" problems on CircleCI when it tries to access the fontawesome repo to download even the free packages.

Force the yarn registry instead, so hopefully we won't attempt to access fontawesome at all.

Reply:
We may be able to remove this at some point in the future, after whatever it is happening in Circle (caches?) adjusts